my_pyscf/mcscf/lasci.py

```diff
@@ -1395,7 +1394,16 @@ def lasci_(self, mo_coeff=None, ci0=None, verbose=None,
 
     state_average = state_average
     state_average_ = state_average_
-    lassi = lassi
+
+    def lassi(self, **kwargs):
+        import warnings
+        lassi_kernel_warn = "Now LASSI have kernel, which takes las instance as input. This [las.lassi()] function " \
+                            "will be removed soon."
+        warnings.warn(lassi_kernel_warn, stacklevel=3)
```

Comment:
I don't think this warning message is necessary at this stage. Even PySCF itself has some functionality like mf.CASSCF, so why shouldn't we keep this at least for a while?